src/StdAssertions.sol

```diff
@@ -146,6 +171,31 @@ abstract contract StdAssertions is DSTest {
         }
     }
 
+    function assertApproxEqAbsDecimal(int256 a, int256 b, uint256 maxDelta, uint256 decimals) internal virtual {
+        uint256 delta = stdMath.delta(a, b);
+
+        if (delta > maxDelta) {
+            emit log("Error: a ~= b not satisfied [int]");
+            emit log_named_decimal_int("  Expected", b, decimals);
+            emit log_named_decimal_int("    Actual", a, decimals);
+            emit log_named_decimal_uint(" Max Delta", maxDelta, decimals);
+            emit log_named_decimal_uint("     Delta", delta, decimals);
+            fail();
+        }
+    }
+
+    function assertApproxEqAbsDecimal(int256 a, int256 b, uint256 maxDelta, uint256 decimals, string memory err)
```

Comment:
Do you mind also adding the equivalents for `assertApproxEqRel` too?

Reply:
done ✅